core/pom.xml

```diff
@@ -34,6 +34,12 @@
             <version>9.0.50</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.tomcat</groupId>
+            <artifactId>tomcat-coyote</artifactId>
+            <version>9.0.50</version>
```

Comment:
On fedora 38 the default package is 9.0.71 and in the spec file the required version was `BuildRequires:    tomcat >= 1:9.0.7` so it works also with newer packages. Should we provide a [range versions](https://docs.oracle.com/middleware/1212/core/MAVEN/maven_version.htm#A1000686) ?